File: testresources/csharp/benchmark/Domain/Direct/Violating/DeclarationVariableWithinForStatement.cs
using Technology.Direct.Dao;
using Domain.Direct;

namespace Domain.Direct.Violating
{
    public class DeclarationVariableWithinForStatement : Base
    {
        public String GetProfileInformation()
        {
    		foreach  (ProfileDAO pdao in profileDAOs) {
    			string p = pdao.toString();
    		}
            return "";
        }
    }
}

File: src/test/resources/java/accuracy/src/domain/direct/violating/DeclarationVariableWithinForStatement.java
package domain.direct.violating;

import domain.direct.Base;
import technology.direct.dao.ProfileDAO;

public class DeclarationVariableWithinForStatement extends Base{
	
	public String getProfileInformation(){
		for (ProfileDAO pdao : profileDAOs) {
			String p = pdao.toString();
		}
		return "";
	}
}